tests/gordo_components/model/test_utils.py

```diff
@@ -3,8 +3,34 @@
 import pytest
 import numpy as np
 import pandas as pd
+import math
 
 from gordo_components.model import utils as model_utils
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.metrics import mean_squared_error
+
+
+def test_metrics_wrapper():
+    # make the features in y be in different scales
+    y = np.array([[1, 1], [2, 2], [3, 3], [4, 4], [5, 5]]) * [1, 100]
+
+    # With no scaler provided it is relevant which of the two series gets an 80% error
+    metric_func_noscaler = model_utils.metric_wrapper(mean_squared_error)
+
+    mse_feature_one_wrong = metric_func_noscaler(y, y * [0.8, 1])
+    mse_feature_two_wrong = metric_func_noscaler(y, y * [1, 0.8])
+
+    assert not math.isclose(mse_feature_one_wrong, mse_feature_two_wrong)
```

Comment:
Very _very_ small note, that you could use `np.isclose(...)`, save you one line. :laughing: